taqueria-plugin-tezos-client/client.ts

```diff
@@ -160,19 +160,21 @@ const preprocessString = (value: string) : string => {
 
 const getSimulateCommand = (opts: Opts, sandbox: Sandbox, sourceFile: string, sourcePath: string) => {
     let storage
```

Comment:
Jimmy, as you know, I'm not a big fan of using `let` as it makes the code harder to read/reason-about.

Here's how I would suggest refactoring this (untested code below):
```ts
const storageGiven  = opts.storage ?? getStorageFromConfig(opts, sourceFile)
const inputGiven = opts.input ? opts.input.toString() : undefined

if (!storageGiven) throw new Error('Error: Please specify a non-empty storage value in the CLI or in the config file.')
if (!inputGiven) throw new Error('Error: Please specify an input value.')

const storage = preprocessString(storageGiven!)
const input = preprocessString(inputGiven)
const entrypoint = opts.entrypoint ? `--entrypoint ${opts.entrypoint}` : ''
const cmd = `docker exec ${sandbox.name} tezos-client run script ${sourcePath} on storage \'${storage}\' and input \'${input}\' ${entrypoint}`

return cmd
```

Reply:
Will refactor this to eliminate `let` 👍